common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerMsalController.java

```diff
@@ -1421,4 +1426,23 @@ private void verifyBrokerVersionIsSupported(@Nullable final Bundle resultBundle,
                     "So, this is not likely a broker version supported issue. Continuing.");
         }
     }
+
+    private void trackTelemetryRegionFromResultBundle(@NonNull final Bundle resultBundle) throws BaseException {
+        final String methodTag = TAG + ":trackTelemetryRegionFromResultBundle";
+        final BrokerResult brokerResult = new MsalBrokerResultAdapter().brokerResultFromBundle(resultBundle);
```

Comment:
Is this a duplicate deserialization? I suspect we unpack the bundle into BrokerResult already somewhere in the code. If yes, then let's just pass BrokerResult here (as opposed to passing a bundle and deserializing again)